tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "lib": [
       "es2017"
     ],
+    "allowJs": true,
```

Comment:
I don't see any `.js`?

Reply:
It makes it easier to pull in new bits of graphlib? It's only there because it was needed at the time.